### ISSUE TITLE
Is v. 0.10.0 Of Custom Markup Lacking Stylesheets?

### ISSUE THREAD

**Issue by TiddlyTweeter** [wikilabs/plugins#66]
This is a MINOR issue easily shown. Enter ...

> `»»» test`

The render is, correctly ...

> `<p class="wltc-l3 wltc">test</p>`

BUT the identing does _not_ occur for me. I am just assuming the stylesheet is missing?

Best wishes
TT

**Comment by pmario** [comment]
Please check out: https://wikilabs.github.io/editions/custom-markup/ V0.10.1

**Comment by TiddlyTweeter** [comment]
> Please check out: https://wikilabs.github.io/editions/custom-markup/ V0.10.1

Its good! Tx! The "**undefined**" class problem has gone! :-).

BUT the issue of the OP goes on with the line ... 

`»»» TEST`

...  as it outputs as ...

![Screenshot 2021-02-27 111650](https://user-images.githubusercontent.com/28830541/109384486-5dba3a00-78ed-11eb-8808-65e64bc7a3d8.jpg)

... with NO indentation. 

I assume the needed stylesheet is just missing from the plugin?

FWIW, the HTML looks right ...

`<p class="wltc-l3 wltc">TEST</p>`

**Comment by pmario** [comment]
```
asdf

»»» TEST
```

It does indent for me. [Stylesheets are listed here](https://wikilabs.github.io/editions/custom-markup/#Predefined%20Stylesheets:%5B%5BPredefined%20Stylesheets%5D%5D%20%3A%2Fplugins%2Fwikilabs%2Fcustom-markup%2Fstyles) ...  AS you can see they are active.

**Comment by pmario** [comment]
>I assume the needed stylesheet is just missing from the plugin?

Ahhh, OK that's right. I wasn't sure, what to include in the plugin. We may discuss this in a different thread or better an issue

**Comment by pmario** [comment]
>I assume the needed stylesheet is just missing from the plugin?

You are right. We need to discuss, which stylesheets should be added to the plugin

**Comment by TiddlyTweeter** [comment]
> We need to discuss, which stylesheets should be added to the plugin

Okay. I guess it depends on the extent to which _you_ want the finished plugin to include such support or not.
I guess that has a lot to do with the target audience--to what extent Custom Markup should be BOTH a "Mark-Up" system and a "Styling" system OR just mainly the "Mark-Up" mechanism??

Speaking personally, I prefer _not_ to have "hard-coded" classes inserted, apart from, maybe "wltc" and actually I don't need stylesheets for production use, rather I'd look at your "demo-site" and take some parts of some of them if I needed such css.

_But maybe you should shart a new discussion on this as it is an important issue--or maybe seek feedback on it during a later beta release with some more people?_

**Comment by pmario** [comment]
Yea, I think it's time for a more public beta. ... But it would need more docs. Can you provide a wiki with your examples, so I can more easily include them?

**Comment by TiddlyTweeter** [comment]
> Yea, I think it's time for a more public beta. ... But it would need more docs. Can you provide a wiki with your examples, so I can more easily include them?

Sure. I try tomorrow to finish enough that it is useful to you.

**Comment by TiddlyTweeter** [comment]
I have been busy in the real world last two days ... but I'm on it now, and try give you materials soon.
